addons/danger/scripts/lambs_danger.fsm

```diff
@@ -230,7 +245,7 @@ class FSM
                         itemno = 9;
                         init = /*%FSM<STATEINIT""">*/"" \n
                          "// keep attacking" \n
-                         "if (_dangerCause in [8, 9] && {_dangerCausedBy call lambs_main_fnc_isAlive}) then {" \n
+                         "if (_dangerCause in [8, 9] && {alive _dangerCausedBy}) then {" \n
```

Comment:
Why the change to just a simple alive check?

Reply:
Two reasons: 
1. Performance (slight)
2. ACE3

ACE3 sets incapacitated units as functionally _invisible_ to the AI. However, that invisibility does not mean the AI isn't being told by its native FSM to try to find and shoot them.  Hence, a simple checks allows our dangerFSM to run _on_ that target-- instead of having them uselessly flail about. 

As it pertains to vehicles, this means a range of interesting and immersive actions.  Such as generally suppressing the location of suspected enemies and _jinking_ out of danger of the same. :)

Reply:
Reverted